docs/blocks/blocks.md

```diff
@@ -8,18 +8,30 @@ Creating a Block
 
-- `hardnessAndResistance` - The hardness controls the time it takes to break the block. It is an arbitrary value. For reference, stone has a hardness of 1.5, and dirt 0.5. If the block should be unbreakable a hardness of -1.0 should be used, see the definition of `bedrock` as an example. The resistance controls the explosion resistance of the block. For reference, stone has a resistance of 6.0, and dirt 0.5.
-- `sound` - Controls the sound the block makes when it is punched, broken, or placed. Requires a `SoundType` argument, see the [sounds][] page for more details.
-- `lightValue` - Controls the light emission of the block. Takes a value from zero to fifteen.
-- `slipperiness` - Controls how slippery the block is. For reference, ice has a slipperiness of 0.98.
+```java
+new Block(Block.Properties.from(Blocks.COBBLESTONE).lightValue(15)).setRegistryName("mymod:myblock");
+```
 
-All these methods are *chainable* which means you can call them in series. See the `Blocks` class for examples of this.
+See `Block.registerBlocks` for more examples on how to create simple blocks.
 
 !!! Note
 
-    Blocks have no setter for their `ItemGroup`. (formerly Creative Tab) This has been moved to the `BlockItem` and is now its responsibility. Furthermore, there is no setter for translation key as it is now generated from the registry name.
+    Blocks have no setter for Item Group (formerly Creative Tab). This has been moved to the ItemBlock, and is now its responsibility. Furthermore, there is no setter for translation key (this is generated based on registry name now).
+
+### `Block.Properties`
+To set the properties of the block, you need to use a `Block.Properties`. There are a couple of options when creating a one. You can use `Block.Properties.from` to copy the properties of an existing block or `Block.Properties.create` to create a new one.
+
+The `Block.Properties` has the following setters:
+
+  - `doesNotBlockMovement` - Makes it so the block does not block movement. Example Usage: plants.
+  - `slipperiness` - Defaults to `0.6F` high values make it more slippery. Example Usage: Ice. **Note:** Vanilla minecraft does not exceed `1.0F` in their blocks slipperiness.
+  - `sound` - Defaults to `SoundType.STONE` see [sounds][] for more information.
+  - `lightValue` - The amount of light emitted by the block. Example Usage: Glowstone. **Note:** this method takes a value from 0 to 15.
+  - `hardnessAndResistance` - Hardness is how long it takes to mine a block (set to -1 for unbreakable). Resistance is explosion resistance.
+  - `needsRandomTick` - Makes the block recieve random ticks. Example Usage: Plants.
+  - `variableOpacity` - If set to true, the game will work out the opacity of your block everytime either `IBlockstate.getOpacity`, `IBlockState.propagatesSkylightDown` or `IBlockState.getLightOpacity` is called. Example Usage: Shulker Boxes.
```

Comment:
`IBlockState` -> `BlockState`. Also, variable opacity doesn't only work out the the opacity. It essentially prevents a cache of the associated `BlockState`s which effects how many times the collision shape is calculated as well.